js/mercado.js

```diff
@@ -12,20 +12,49 @@ module.exports = class mercado extends Exchange {
         return this.deepExtend (super.describe (), {
             'id': 'mercado',
             'name': 'Mercado Bitcoin',
-            'countries': [ 'BR' ], // Brazil
+            'countries': ['BR'], // Brazil
             'rateLimit': 1000,
             'version': 'v3',
             'has': {
                 'CORS': true,
-                'createMarketOrder': false,
+                'createMarketOrder': true,
                 'fetchOrder': true,
                 'withdraw': true,
+                'fetchOHLCV': true,
+                'fetchOrders': true,
+                'fetchOpenOrders': true,
+                'fetchTicker': true,
+                'fetchTickers': true,
+            },
+            'timeframes': {
+                '5m': '5m',
+                '15m': '15m',
+                '30m': '30m',
+                '1h': '1h',
+                '6h': '6h',
+                '12h': '12h',
+                '1d': '1d',
+                '3d': '3d',
+                '1w': '1w',
+                '2w': '2w',
+            },
+            'timeframesMilliseconds': {
+                '5m': 300,
+                '15m': 900,
+                '30m': 1800,
+                '1h': 3600,
+                '6h': 21600,
+                '1d': 86400,
+                '3d': 64800,
+                '1w': 604800,
+                '2w': 1209600,
             },
             'urls': {
                 'logo': 'https://user-images.githubusercontent.com/1294454/27837060-e7c58714-60ea-11e7-9192-f05e86adb83f.jpg',
                 'api': {
                     'public': 'https://www.mercadobitcoin.net/api',
                     'private': 'https://www.mercadobitcoin.net/tapi',
+                    'v4Public': 'https://www.mercadobitcoin.com.br/v4',
```

Comment:
Can you share the docs for their v4?

Reply:
The version 4 was to created only to export the OHLCV. I shared in code the docs using the urls bellow:

https://www.mercadobitcoin.com.br/api-doc
https://www.mercadobitcoin.com.br/trade-api